proposals/drafts/campaign_finance_filings.rst

```diff
@@ -104,21 +104,17 @@ Filing
 id
     Open Civic Data-style id in the format ``ocd-cf-filing/{{uuid}}``
 
-original_id
+identifiers
     **optional**
-    In some jurisdictions, the original jurisdictionally-assigned ID of a Filing
-    may be meaningful, so preserve it here.
+    Upstream ids of the disclosure if any exist, such as the filing ID assigned by the Senate Office of Public Record
 
-type
+classification
     **optional**
     Filing Type (jurisdiction-specific)
 
 committee
     Committee making the Filing.
 
-first_filing_date
```

Comment:
Maybe. To me it seems more like a special case, since filings are often required by certain deadlines, and can be amended after those deadlines. So it's useful to know what the "first" filing date is. Which I suppose is really just accomplished by looking at the first filing_action present, but maybe there's enough utility here that it's worth keeping it separate.